hoomd/md/many_body.py

```diff
@@ -0,0 +1,402 @@
+import hoomd
+from hoomd.md import _md
+from hoomd.md.pair import Pair
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+
+
+class Triplet(Pair):
```

Comment:
Why does this inherit from Pair? This isn't an is-a relationship. The only commonalities are that both use neighbor lists and have r_cut type parameters. I think this should ineherit from `force.Force`.

Reply:
This shows up again in later comments where the inherited `r_on`, `mode` and `compute_energy` are not present in the C++ implementation of triplet potentials.

Reply:
I think I did this for the purpose of code re-use, back when I mistakenly thought there was an `r_on` for triplet potentials.